airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/CdcMssqlSslSourceTest.java

```diff
@@ -18,8 +18,12 @@
 import java.util.Map;
 import javax.sql.DataSource;
 import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
 
-@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@TestInstance(Lifecycle.PER_METHOD)
```

Comment:
curious why do we need new instance of the class for each inner test

Reply:
just to allow parallelism. Those tests are not particularly fast